### PR TITLE
Fix append branch (HPC-10585)

### PR DESCRIPTION
**Pull request by itkovian**

### CODE REVIEW

File: lib/vsc/filesystem/quota/ktools.py [hunk 1]
```diff
@@ -266,9 +266,9 @@ def process_event(self, event, dry_run):
             if cached_usage != event:
```
Comment (stdweird) [inline lib/vsc/filesystem/quota/ktools.py]: please invert the bloks and logic

Comment (itkovian) [reply lib/vsc/filesystem/quota/ktools.py]: Inversion applied.

**Comment by stdweird** [comment]
@itkovian diff on this PR shows no version bump

**Comment by itkovian** [comment]
Odd. It does now though.

**Pull request merged**